frontend/src/components/MainNavigation.vue

```diff
@@ -295,13 +294,26 @@ export default {
       return this.isProjectNameMatchingFilter(projectName)
     },
     highlightedProject () {
-      return this.sortedAndFilteredProjectListWithAllProjects[this.highlightedProjectIndex]
+      if (!this.highlightedProjectName) {
```

Comment:
if we only need it for navigateToProject it should be a method not a computed prop